bazel_tools/scala-escape-jvmflags.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/java/io/bazel/rulesscala/exe/LauncherFileWriter.java b/src/java/io/bazel/rulesscala/exe/LauncherFileWriter.java
+index f54a578..0d871ef 100644
+--- a/src/java/io/bazel/rulesscala/exe/LauncherFileWriter.java
++++ b/src/java/io/bazel/rulesscala/exe/LauncherFileWriter.java
+@@ -36,7 +36,8 @@ public class LauncherFileWriter {
+             .addKeyValuePair("jar_bin_path", jarBinPath)
+             .addKeyValuePair("java_start_class", javaStartClass)
+             .addKeyValuePair("classpath", classpath)
+-            .addJoinedValues("jvm_flags", " ", jvmFlags)
++            .addKeyValuePair("escape_jvmflags", "0")
++            .addJoinedValues("jvm_flags", "\t", jvmFlags)
```

Comment:
@aherrmann-da There's a change in separator used for `jvm_flags` as well: https://github.com/laszlocsomor/bazel/commit/569e2cb4d1b7653b5a02bbf664765476233e6c33 - this should resolve the most recent problem

Reply:
@majcherm-da Thanks for fixing this! The very latest Windows failure looks like flakiness. A download failed. Restarting.